guacamole-docker/bin/start.sh

```diff
@@ -687,6 +687,18 @@ END
     ln -s /opt/guacamole/cas/guacamole-auth-*.jar   "$GUACAMOLE_EXT"
 }
 
+##
+## Adds properties to guacamole.properties which configure the json
+## authentication provider.
+##
+associate_json() {
+    # Update config file
+    set_property "json-secret-key"                 "$JSON_SECRET_KEY"
```

Comment:
While we're at it, should we also make the optional property "json-trusted-networks" available, here?

Reply:
Good point, that property wasn't documented and I wasn't aware it existed. Should be fixed now